openstack/identity/v3/extensions/tokens/errors.go

```diff
@@ -0,0 +1,72 @@
+package tokens
```

Comment:
yes, they could be reused, i thought it would be more neat!, but yes seems reasonable.